cli/tests/unit_node/testdata/child_process_stdio.js

```diff
@@ -3,7 +3,7 @@ import process from "node:process";
 import * as path from "node:path";
 
 const script = path.join(
-  path.dirname(path.fromFileUrl(import.meta.url)),
+  path.dirname(new URL(import.meta.url).pathname),
```

Comment:
This won't work on Windows. These can be changed to use `fileURLToPath` from `node:url`

Reply:
thanks! I made the update